src/app/scripts/cac/control/cac-control-explore.js

```diff
@@ -296,8 +299,6 @@ CAC.Control.Explore = (function (_, $, Geocoder, MapTemplates, HomeTemplates, Ro
         if (exploreLatLng && tabControl.isTabShowing(tabControl.TABS.EXPLORE)) {
             clickedExplore();
         }
-
-        getNearbyPlaces();
```

Comment:
Calling it as part of `setActiveTab` in the home controller instead seems sufficient and results in no extra calls. Modifying the throttle behavior would just mask an extraneous call being made and introduce some extra delay.